### PR TITLE
Add config for animation_with_microtasks test

### LINKED ISSUE #87345: Add animation_with_microtasks_perf_ios__timeline_summary to framework benchmarks
See: https://github.com/flutter/flutter/issues/86802

### PR DESCRIPTION
**Pull request by iskakaushik**
fixes: https://github.com/flutter/flutter/issues/87345
Also see: https://flutter-review.googlesource.com/c/infra/+/16380

### CODE REVIEW

File: .ci.yaml [hunk 1]
```diff
@@ -1816,6 +1816,15 @@ targets:
         ["devicelab"]
     scheduler: luci
 
+  - name: Mac_ios animation_with_microtasks_perf_ios__timeline_summary
+    builder: Mac_ios animation_with_microtasks_perf_ios__timeline_summary
+    presubmit: false
+    bringup: true
```
Comment (CaseyHillers) [inline .ci.yaml:1822]: Can you add a comment linking to the github issue? It's helpful to know this is in bringup versus flaky

**Review by CaseyHillers — APPROVED** [review]

**Comment by CaseyHillers** [comment]
You'll need to add this test to [TESTOWNERS](https://github.com/flutter/flutter/blob/master/TESTOWNERS)

\cc @keyonghan as fyi

File: .ci.yaml (hunk 1 above)
Comment (CaseyHillers) [reply .ci.yaml:1825]: Thanks, lgtm

**Review by keyonghan — APPROVED** [review]
LGTM

**Pull request merged**